### ISSUE TITLE
Does the model have a validation set

### ISSUE THREAD

**Issue by TobyTongg** [thograce/BGNet#6]
I only saw the training set and test set in the project. Does the model training not require validation set? Looking forward to your reply.

**Comment by thograce** [comment]
> I only saw the training set and test set in the project. Does the model training not require validation set? Looking forward to your reply.

No validation set. You can look at other COD models, such as the baseline model SINet(https://github.com/DengPingFan/SINet ). It also seems that there is no explicit validation set.

**Comment by TobyTongg** [comment]
> > 我只看到了项目中的训练集和测试集。模型训练不需要验证集吗？期待您的回复。
> 
> 没有验证集。您可以查看其他 COD 模型，例如基线模型 SINet（https://github.com/DengPingFan/SINet）。似乎也没有明确的验证集。

Got it. Thank you
